include/seqan3/alphabet/union_alphabet.hpp

```diff
@@ -107,15 +168,21 @@ constexpr auto at_least_type() {
  * the gap alphabet). The alphabet may be brace initialized from the static letter members (see above).
  * Note that you cannot assign regular characters, but additional functions for this are available.
  *
+ *~~~~~~~~~~~~~~~{.cpp}
  *     union_alphabet<dna4, gap> my_letter{};
+ *     union_alphabet<dna4, gap> converted_letter = dna4::C;
  *     // doesn't work:
  *     // union_alphabet<dna4, gap> my_letter{'A'};
  *
  *     union_alphabet<dna4, gap>.from_char('C'); // <- this does!
  *
  *     union_alphabet<dna4, gap>.from_char('-'); // gap character
+ *
+ *     union_alphabet<dna4, gap>.from_char('K'); // unknown characters map to the default/unknown
+ *                                               // character of the first alphabet type (i.e. A of dna4)
  *     if (my_letter.to_char() == 'A')
  *        std::cout << "yeah\n"; // "yeah";
+ *~~~~~~~~~~~~~~~
  */
 
 template <typename first_alphabet_type, typename ...alphabet_types>
```

Comment:
https://github.com/h-2/seqan3/blob/feature/concatenated_sequence/include/seqan3/container/concatenated_sequences.hpp#L54

(also add the \brief line to stuff if it has more than one line of documentation)